### DISCUSSION TITLE
Wow, the document is so beautifully written.

### DISCUSSION THREAD

**Discussion by LOUSANPANG** [primefaces/community#716] [PrimeVue]
Hello, congratulations 🎉. Sorry, I just noticed such a good UI repository. I would like to ask if there is a streamlined version of this document? I really want to use it~

**Comment by FlipWarthog** [comment]
Can you elaborate? Which document are you referring to?

**Reply by LOUSANPANG** [reply]
[PrimeVue UI](https://primevue.org/installation/) Is there a streamlined version?

**Reply by FlipWarthog** [reply]
I'm not sure what you mean by that.
